models/aeif_cond_alpha.h

```diff
@@ -84,51 +86,51 @@ This implementation uses the embedded 4th order Runge-Kutta-Fehlberg solver with
 adaptive step size to integrate the differential equation.
 
 The membrane potential is given by the following differential equation:
-C dV/dt= -g_L(V-E_L)+g_L*Delta_T*exp((V-V_T)/Delta_T)-g_e(t)(V-E_e)
-                                                     -g_i(t)(V-E_i)-w +I_e
+@f[ C dV/dt= -g_L(V-E_L)+g_L*Delta_T*exp((V-V_T)/Delta_T)-g_e(t)(V-E_e)
```

Comment:
should we switch the equations to full LaTeX style? (``\Delta``) for instance? And possibly fractions, or should we keep it as readable as possible in the raw file?

Reply:
In the NESTML model documentation, we have decided to move towards full LaTeX (as part of reStructuredText). For now, we have not yet addressed converting all the existing documentation, but any new documentation added should conform to this format.

Reply:
yeah, this sounds good, I don't see why we should trade off documentation readability to favor the raw file

Reply:
I agree, full LaTeX would be great!

Reply:
+1 for full LaTeX.

Also, I saw that some of the equations are now wrongly rendered. E.g. `tau_max` in the [docs of the correlation_detector`](https://nest-test.readthedocs.io/en/extract_models_breathe/models/devices.html#_CPPv3N4nest20correlation_detectorE) should be "tau<sub>max</sub>" and currently is "tau<sub>m</sub>ax". I thus think we should fix documentation rather soon (either in this PR or very quickly after)

Reply:
I went through and updated the model formulas to convert them to full latex. Please double check that they are correct.  https://nest-test.readthedocs.io/en/extract_models_breathe/models/neurons.html

Reply:
Thanks for the update! I changed a few things to make fractions more readable and make parameters names in the equation coherent with the C++ counterparts